python/tk_multi_workfiles/actions/open_publish_actions.py

```diff
@@ -51,13 +51,13 @@ def execute(self, parent_ui):
 class ContinueFromPublishAction(ContinueFromFileAction):
     """
     """
-    def __init__(self, file, file_versions, environment):
+    def __init__(self, file, file_versions, environment, next_version_override):
         """
         """
         # This command does not pass down a next_version because it is assumed
         # that the publishes have all been retrieved from Shotgun, so therefore
         # the default behaviour from the base class of doing max + 1 works great.
-        ContinueFromFileAction.__init__(self, "Continue Working From Publish", file, file_versions, environment)
+        ContinueFromFileAction.__init__(self, "Continue working from Publish in your Work Area", file, file_versions, environment, next_version_override)
```

Comment:
are we capitalizing "Work Area" in other places? If not, i suggest we keep it lower case here.

Reply:
Yeah, we are.